.github/workflows/test-ios-e2e.yml

```diff
@@ -31,21 +34,21 @@ jobs:
           base64 -d <<< ${{ secrets.CONFIG }} > Config.xcconfig
           base64 -d <<< ${{ secrets.TEST_CONFIG }} > ConfigTest.xcconfig
 
-      - name: Run tests on iPad
-        working-directory: ./LocationServices
-        run: bundle exec fastlane run_e2e_tests device:"iPad Pro (12.9-inch) (6th generation)"
+      # - name: Run tests on iPad
```

Comment:
why e2e test is disabled?

Reply:
E2E is disable for this specific device since the latest xcode runner beta is not available for this device. Also, to save time. Otherwise it would double the time.

Reply:
> E2E is disabled for this specific iPad device since the latest xcode runner beta is not available for this device.
I don't quite get what does it mean. Does it mean the feature is no longer work on ipad. 

I wouldn't mind to have test run longer if it mean that all platforms are covered.

Reply:
I don't see any e2e test workflow triggered in PR. What is the plan for it?

Reply:
This PR will not merge into main branch. After you merge this PR. The E2E will trigger on this main PR branch: https://github.com/aws-geospatial/amazon-location-features-demo-ios/pull/51

This is due to the same reason (the secret issue) we have for web & android

Reply:
This PR makeen-project:fixes/E2ETest-cases-fixes -> merge into -> aws-geospatial:E2E_test_case_main

And this PR https://github.com/aws-geospatial/amazon-location-features-demo-ios/pull/51 
aws-geospatial:E2E_test_case_main -> run tests -> merge into -> main branch

Reply:
ic, then based on the process, Dan or Harshit can merge this CR. I'll take a look at #51. Thanks for explaining.